### PR TITLE
Install OpenMusic by default in Linux

### PR DESCRIPTION
**Pull request by FifoIronton**
In order to load parks using certain music files, such as the dodgems theme, the Linux build must include OpenMusic by default.  
  
Note that this isn't a long-term fix, and follow-up work to the [download.cmake](https://github.com/OpenRCT2/OpenRCT2/blob/develop/cmake/download.cmake) file to allow both OpenMusic and OpenSoundEffects to be imported at the same time. The `download_openrct2_zip` function expects each downloaded zip file to be in its own unique directory, while OpenMusic and OpenSoundEffects are both imported into the `install/usr/share/openrct2/` directory

See https://github.com/OpenRCT2/OpenRCT2/issues/19160 for additional info

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
@janisozaur are you able to review this one.

**Comment by ZehMatt** [comment]
ping @janisozaur

**Review by janisozaur — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -158,6 +160,11 @@ if (MACOS_USE_DEPENDENCIES)
     endif ()
 endif ()
 
+# If OS is Linux, import OpenSoundEffects and OpenMusic
+if(UNIX AND NOT APPLE)
+    set(DOWNLOAD_OPENMSX ON)
```
Comment (tupaschoal) [inline CMakeLists.txt:165]: Since this PR disabled `OPENSFX` and `OPENMSX`, shouldn't it have reenabled both according to this if's comment?

**Comment by qwertychouskie** [comment]
This appears to break the PPA builds (https://launchpad.net/~openrct2/+archive/ubuntu/nightly).  Launchpad doesn't allow build runners full Internet access.  A graceful fallback in the event of no Internet access during compilation should be added.

**Pull request merged**